packages/image_picker/image_picker/README.md

```diff
@@ -23,7 +23,6 @@ As a result of implementing PHPicker it becomes impossible to pick HEIC images o
 Add the following keys to your _Info.plist_ file, located in `<project root>/ios/Runner/Info.plist`:
 
 * `NSPhotoLibraryUsageDescription` - describe why your app needs permission for the photo library. This is called _Privacy - Photo Library Usage Description_ in the visual editor.
-  * This permission is not required for image picking on iOS 11+ if you pass `false` for `requestFullMetadata`.
```

Comment:
LGTM if you want to remove it fully, but maybe instead we should leave it but change to something like "This permission will not be requested if you always pass `false` for `requestFullMetadata`, but App Store policy requires including the plist entry."

That way people (including future us, potentially) won't think it's a mistake that we're listing it here when they read the code documentation that talks about how that flag interacts with permissions.